### PR TITLE
added ecom card

### PR DESCRIPTION
**Pull request by shivanigangadharan**

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -13,6 +12,9 @@
     font-weight: 500;
     margin: 20px 0px 0px 0px;
 }
+.img{
+    max-height: 300px;
```
Comment (abhishektanwar) [inline components/card.css:16]: maybe use rems here?

File: components/card.css [hunk 2]
```diff
@@ -1,8 +1,7 @@
 .container-card{
     background: ghostwhite;
     border-radius: 3px;
-    width: 350px;
-    padding: 0px 0px 10px 0px;
+    max-width: 300px;
```
Comment (abhishektanwar) [inline components/card.css:4]: use rems here?

File: components/card.css [hunk 3]
```diff
@@ -23,6 +25,12 @@
 .content{
     margin: 20px 20px 0px 20px;
```
Comment (abhishektanwar) [inline components/card.css:26]: rem could be used here .

**Review by abhishektanwar — COMMENTED** [review]
Good to merge . Please have a look at the comments.

**Review by dev-enforced — COMMENTED** [review]
The code looks good to me!!The utility classes also are used

File: components/card.css (hunk 2 above)
Comment (shivanigangadharan) [reply components/card.css:4]: done

File: components/card.css (hunk 3 above)
Comment (shivanigangadharan) [reply components/card.css:26]: Generally I'd use rem for >20px, noted though. thanks

File: components/card.css (hunk 1 above)
Comment (shivanigangadharan) [reply components/card.css:16]: done.

**Pull request merged**